pyproject.toml

```diff
@@ -98,6 +98,7 @@ dev = [
     "sphinx",
     "sphinx_rtd_theme",
     "sphinxcontrib-spelling < 8.0.1",
+    "sphinx_js",
```

Comment:
Hi Alec,
please have a look at this comment I made regarding the re-addition of sphinx-js:
https://github.com/Dallinger/Dallinger/issues/4992#issuecomment-2561265595  

I think there has to be some cleanup to be done.

Reply:
@fmhoeger I believe the `sphinx-js` change is working on this branch (the `constraints.txt` now pins `sphinx-js==4.0.0`). We could add `sphinx-js >= 4.0.0` to `pyproject.toml` to ensure an older version isn't pulled in, but I don't think it's necessary. Let me know if there's anything else missing here.

Reply:
It should be fine like it is. :+1:

Reply:
In our patched `markupsafe` setup I had to install `nodejs` and `jsdocs` in order for the docs to be able to be built. See my commit here: https://github.com/Dallinger/Dallinger/pull/6904/files

We can probably remove lines 6-9 again. What do you think?

Reply:
@fmhoeger I think those are still needed to successfully run sphinx with JSDoc/sphinx-js support. We're using the same sphinx-js code we were when we were on the patched branch, it's just been released to PyPI now, so I don't think anything else besides the pin needs to change.

Reply:
OK, I see. Yes, just leave them then. :+1: